src/components/textInput.re

```diff
@@ -91,6 +92,26 @@ let make =
                 },
               autoCapitalize,
             ),
+          "autoComplete":
+            UtilsRN.option_map(
+              x =>
+                switch (x) {
```

Comment:
Please use `autocomplete->Belt.Option.map(fun` instead of UtilsRN (which is going to be removed, see #324)